cloudflare/src/index.ts

```diff
@@ -16,29 +16,32 @@ interface EnvWithCustomVariables extends Env {
 
 export class MyContainer extends Container<EnvWithCustomVariables> {
   constructor(ctx: DurableObjectState, env: EnvWithCustomVariables) {
-        super(ctx, env);
-
-        // Port the container listens on (default: 8080)
-        this.defaultPort = 8080;
-
-        this.envVars = {
-          AWS_ACCESS_KEY_ID: env.AWS_ACCESS_KEY_ID,
-          AWS_SECRET_ACCESS_KEY: env.AWS_SECRET_ACCESS_KEY,
-          R2_BUCKET: env.R2_BUCKET,
-          HTSGET_LOCATIONS: env.HTSGET_LOCATIONS,
-          AWS_REGION: "auto", // Otherwise S3 sdk will error out: ResolveEndpointError { message: "A region must be set when sending requests to S3. }. Can also be safely set to 'us-east-1'"
-          RUST_LOG: 'trace,htsget_lambda=trace,htsget_lambda=trace,htsget_config=trace,htsget_http=trace,htsget_search=trace,htsget_test=trace',
-          LOG_SENSITIVE_BODIES: 'false', // aws-sdk-rust will show ALL trace info
-          LOG_SIGNABLE_BODY: 'false'     // will log sensitive data
-        };
-    }
+    super(ctx, env);
+
+    // Port the container listens on (default: 8080)
+    this.defaultPort = 8080;
+
+    this.envVars = {
+      AWS_ACCESS_KEY_ID: env.AWS_ACCESS_KEY_ID,
+      AWS_SECRET_ACCESS_KEY: env.AWS_SECRET_ACCESS_KEY,
+      R2_BUCKET: env.R2_BUCKET,
+      HTSGET_LOCATIONS: env.HTSGET_LOCATIONS,
+      AWS_REGION: "auto", // Otherwise S3 sdk will error out: ResolveEndpointError { message: "A region must be set when sending requests to S3. }. Can also be safely set to 'us-east-1'"
+      RUST_LOG:
```

Comment:
@mmalenic Would you mind commenting this one? I don't think a prod deploy needs `trace` loglevel?